src/main/java/core/basesyntax/service/writetofileservice/impl/WriteToCsvFileServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.writetofileservice.impl;
+
+import core.basesyntax.service.writetofileservice.WriteToFileService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+
+public class WriteToCsvFileServiceImpl implements WriteToFileService {
+    @Override
+    public void writeDataToFile(Map<String, Integer> map) {
```

Comment:
To follow single responsibility principle please extract logic that creates report from service that should only write report
```suggestion
    public void writeDataToFile(String report) {
```
or
```suggestion
    public void writeDataToFile(List<String> report) {
```